packages/react-app/theme.js

```diff
@@ -1,5 +1,14 @@
 import { extendTheme } from "@chakra-ui/react";
 
+const Card = {
+  baseStyle: props => ({
+    border: "1px solid",
+    borderColor: props.colorMode === "light" ? "light.text" : "dark.text",
+    background: props.colorMode === "light" ? "light.base" : "dark.base",
+    boxShadow: `8px 8px 0px 0px ${props.colorMode === "light" ? "#808CFF" : "var(--chakra-colors-dark-baseBlue)"}`,
+  }),
+};
```

Comment:
Don't go too crazy about this (only do it if you are having fun / learning haha). 

Using props in <Box> is good too :P

Reply:
Yeah lol sorry, was just surfing through docs and found this example related to our use case : 
https://arc.net/l/quote/pjlmouys

So thought of using it, since we are using kind of similar structure for Cards